### ISSUE TITLE
Build pip package from source failed

### ISSUE THREAD

**Issue by jrabary** [tensorflow/tensorflow#1041]
I got the following error while building pip package from the master version 

`tensorflow/core/BUILD:75:1: Target '//google/protobuf:protobuf_python_genproto' is not visible from target '//tensorflow/core:protos_all_py_genproto'. Check the visibility declaration of the former target if you think the dependency is legitimate.`

Bazel version 0.1.4 and 0.1.5

**Comment by vrv** [comment]
@damienmg: any ideas here?  Did something change about visibility targets?

**Comment by Dringite** [comment]
What Bazel version should we use? The site says 0.1.1

**Comment by vrv** [comment]
If you are building from HEAD, you need to look at the 'master' version of the docs, not 0.6.0.

Which says:

"Follow instructions here to install the dependencies for bazel. Then download the latest stable bazel version using the installer for your system and run the installer as mentioned there"

And currently bazel stable is 0.1.5.

(Bazel is evolving very quickly, along with us, so there's some unavoidable churn at the moment).

**Comment by Dringite** [comment]
Successfully built bleeding edge TensorFlow pip package using 0.1.4 just now.

**Comment by mondatu** [comment]
Does anyone have this problem and manage to fix it? I used both bazel-0.1.5-installer-linux-x86_64.sh and bazel-0.1.4-installer-linux-x86_64.sh with no luck. The version of Tensorflow I tried is c4207090d68151fb967672a90ea6d574e62cfba0 (the latest master version at the moment)

**Comment by martinwicke** [comment]
What version of protobuf do you have? Did you accidentally move the
submodule ref in the tensorflow repo to something too new or too old?

**Comment by jrabary** [comment]
I finally manage to build it with a fresh clone of the repository. Maybe I had an old build artifact that broke the build. My settings : bazel 0.1.5, python 3.4

**Comment by martinwicke** [comment]
I think the submodule pointer to protobuf may have been confused. Closing.

**Comment by mondatu** [comment]
@martinwicke 
This is my protobuf version:
git submodule status 
+55ad57a235c009d0414aed1781072adda0c89137 google/protobuf (v3.0.0-alpha-4-179-g55ad57a)
I didn't recall moving any submodule ref. When I reverted to some previous Tensorflow version (on Feb 8), e.g., df66b9fe049cb17e58454f309b662bdaf0d14fdb, I didn't have any problem with compiling.

I did a fresh clone as  @jrabary did and it works fine now. Thanks, mates!
